<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1866</width>
         <height>931</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QLabel" name="foregroundLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>531</width>
          <height>421</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>forground</string>
        </property>
       </widget>
       <widget class="QLabel" name="bodyPartLabel">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>10</y>
          <width>581</width>
          <height>491</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>bodyPartClassification</string>
        </property>
       </widget>
       <widget class="QLabel" name="skeletonLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>460</y>
          <width>640</width>
          <height>480</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>551</y>
          <width>761</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label5">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>bodyPartClassification</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_bodyPartClassification">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label6">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>centroid</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_centroid">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>skeleton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_skeleton">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>smooth</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_smooth">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_totalCaculation1">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>totalCaculation</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_totalCaculation">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label1">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>imageLoad</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_imageLoad">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label2">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>fps</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_fraps">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
             <italic>false</italic>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="depthLabel">
        <property name="geometry">
         <rect>
          <x>1200</x>
          <y>10</y>
          <width>530</width>
          <height>431</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>foregroundLabel</zorder>
       <zorder>skeletonLabel</zorder>
       <zorder>layoutWidget</zorder>
       <zorder>depthLabel</zorder>
       <zorder>bodyPartLabel</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
